feat: introduction & No About page (#21)
KILTprotocol/docs: docs/apps/00-introduction.md

title: Introduction
---

> This section provides help with the various tools and apps that can be used to interact with the KILT chain.

Reviewer: ```suggestion
This section details the various tools and apps that can be used to interact with the KILT chain.
```

> * [The Parity Signer](./01-parity-signer.md) can be used to create offline transactions. It is currently not recommended to use this app for using functionality like attesting and revoking claims.

Reviewer: ```suggestion
* [The Parity Signer](./01-parity-signer.md) can be used to create offline transactions. It is currently not recommended to use the app for using functionality like attesting and revoking claims.
```

> * The [Polkadot Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.api.onfinality.io%2Fpublic-ws#/explorer) can be used to interact with the KILT chain on a very low level and targets experienced users

Reviewer: ```suggestion
* The [Polkadot Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fspiritnet.api.onfinality.io%2Fpublic-ws#/explorer) can be used to interact with the KILT chain on a very low level and targets experienced users.
```

Reviewer: Also, we should mention that they are connecting via `onFinality`.
